src/app/app-extras.module.ts

```diff
@@ -62,7 +66,8 @@ import {
     }
   ],
   entryComponents: [
```

Comment:
`entryComponents` are [deprecated](https://angular.io/guide/entry-components).

Reply:
Good point, but since we allow for Ivy to be turned off we do still need them.